### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by HarshithaKonduru214**
Please review my pr.
It includes - signup, login and logout

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]
Code looks clean 
Good folder structure 
Classnames are self explanatory

File: src/Context/AuthContext.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    const signInUser = async (email, password) => {
+        try {
+          const {
+            data: { foundUser, encodedToken },
+            status,
+          } = await axios.post("/api/auth/login", {
+            email: email,
+            password: password,
+          });
+          console.log(status)
```
Comment (anupkgurung) [inline src/Context/AuthContext.js:50]: you can remove logs before committing the code.

**Review by anupkgurung — APPROVED** [review]
code is clean.
nice use of components

File: src/Context/AuthContext.js (hunk 1 above)
Comment (HarshithaKonduru214) [reply src/Context/AuthContext.js:50]: Thanks for pointing it out. Removed it.

**Pull request merged**